.github/workflows/ci.yml

```diff
@@ -49,13 +49,13 @@ jobs:
         run: yarn --frozen-lockfile
 
       - name: Build packages
-        run: yarn build
+        run: yarn build --filter=[main...${{ github.head_ref }}]
 
       - name: Lint
-        run: yarn lint
+        run: yarn lint --filter=[main...${{ github.head_ref }}]
 
       - name: Type check
-        run: yarn type-check
+        run: yarn type-check --filter=[main...${{ github.head_ref }}]
 
       - name: Test
-        run: yarn test
+        run: yarn test --filter=[main...${{ github.head_ref }}]
```

Comment:
```
Run yarn build --filter=[main...fix-turborepo]
yarn run v1.22.19
$ turbo run build --filter=[main...fix-turborepo]
 ERROR  run failed: failed to resolve packages to run: finding changes relative to /home/runner/work/polaris/polaris: exit status 128
Turbo error: failed to resolve packages to run: finding changes relative to /home/runner/work/polaris/polaris: exit status 128
error Command failed with exit code 1.
```

@samrose3 this is working great! If there aren't any changes we need to skip the check instead of exit on error.